docs/openapi/open-api.yml

```diff
@@ -69,3 +138,68 @@ components:
         - mSettings
         - mStandings
         - mTeam
+        - players_wl
+        - proTeamSchedules_wl
+        - mDraftDetail
+    
+    TransactionPayload:
+      description: Payload for POST request body to change a lineup
+      type: "object"
+      required:
+        - isLeagueManager
+        - teamId
+        - type
+        - memberId
+        - scoringPeriodId
+        - executionType
+        - items
+      properties:
+        isLeagueManager: 
+          description: not sure if this can be boolean and not sure what happens if its true
```

Comment:
I've been just leaving the description tag out if I'm not sure what it means. I'm not sure if that is the best approach to where we aren't sure or if its better to explicitly call out when we don't know what it means. Thoughts?

Reply:
works for me, I can try and get a league manager to test it

Reply:
I had been leaving notes/thoughts for future reference while we figure it out